tests/end2end/data/userscripts/open_current_url.bat

```diff
@@ -0,0 +1 @@
+echo open -t %QUTE_URL% >> "%QUTE_FIFO%"
```

Comment:
nitpick, please add a final newline (but no blank line) at the end of the file :wink: